daml-lf/engine/src/main/scala/com/digitalasset/daml/lf/engine/Result.scala

```diff
@@ -143,7 +146,7 @@ object Result {
     ResultNeedPackage(
       packageId,
       {
-        case None => ResultError(Error(s"Couldn't find package $packageId"))
+        case None => ResultError(Error.Interpretation.Generic(s"Couldn't find package $packageId"))
```

Comment:
this is really a fatal error right? we should never hit this afaict. (perfectly fine to not touch in this PR)

Reply:
Was planned for another  PR.